source/ui/beamline/AMXRFDetailedDetectorView.h

```diff
@@ -48,13 +48,6 @@ class AMXRFDetailedDetectorView : public AMXRFBaseDetectorView
 	/// Re-implementing but still going to use the base class buildDetectorView since this view is merely adding to it.
 	virtual void buildDetectorView();
 
-	/// Returns the energy range used for filtering.  If no range has been provided the range is null.
-	const AMRange &energyRange() const { return emissionLineValidator_->range(); }
```

Comment:
Why were these removed?  Were they moved somewhere else and then not removed initially?